README.md

```diff
@@ -83,6 +83,62 @@ npm run dev
 
 To try out integrations with test card numbers and payment method details, see [Test card numbers](https://docs.adyen.com/development-resources/test-cards/test-card-numbers).
 
+## Testing webhooks
+
+Webhooks deliver asynchronous notifications (like payment status) and it is important to test them during the setup of your integration. You can find more information about webhooks in [this detailed blog post](https://www.adyen.com/blog/Integrating-webhooks-notifications-with-Adyen-Checkout).
+
+This sample application provides a simple webhook integration exposed at `/api/webhooks/notifications`. For it to work, you need to:
+
+1. Provide a way for the Adyen platform to reach your running application
+2. Add a Standard webhook in your Customer Area
+
+### Making your server reachable
+
+Your endpoint consuming the incoming webhook notifications must be accessible by Adyen (ie public access).
```

Comment:
Rephrase for simplicity: `Your endpoint that will consume the incoming webhook notifcations must be publicly accessible by Adyen`

Reply:
I see, here we should either say `publicly accessible` or `accessible by Adyen`, together (`publicly accessible by Adyen`) it doesn't sound correct

Reply:
Makes sense. How about: "Your endpoint that will consume the incoming webhook should be publicly accessible."

Reply:
Agreed